bin/ftdi_reader.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+# This file is part of ts_ess_sensors.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+import logging
+import os
+import time
+
+from pylibftdi import Device
+
+logging.basicConfig(
+    format="%(asctime)s:%(levelname)s:%(name)s:%(message)s",
+    level=logging.DEBUG,
+)
+
+BAUDRATE: int = 19200
+DELIMITER: str = ","
+TERMINATOR: str = "\r\n"
+
+device_id = "AL05OBVR"
+
+device = Device(device_id, mode="t", lazy_open=True, auto_detach=False)
+device.open()
+device.baudrate = BAUDRATE
+if not device.closed:
+    logging.info(f"VCP {device_id} open.")
+    device.flush()
+else:
+    logging.info("Failed to open VCP.")
+    raise IOError(f"VcpFtdi:{device_id}: Failed to open VCP.")
+
+for i in range(5):
+    resp: str = ""
+    while not resp.endswith(TERMINATOR):
+        resp += device.read(1)
+    print(resp.strip())
+
+device.close()
+if device.closed:
+    logging.info(f"VCP {device_id} closed.")
+else:
+    logging.info("Failed to close VCP.")
+    raise IOError(f"VcpFtdi:{device_id}: Failed to close VCP.")
+
+os.system("sudo rmmod ftdi_sio")
+os.system("sudo rmmod usbserial")
+os.system("sudo modprobe ftdi_sio")
```

Comment:
Unfortunately the `sudo` lines are necessary since the RPi kernel doesn't allow for automatic resetting of the FTDI devices. However, this script is merely for testing of the hardware and shouldn't be part of the project. I moved it to a private (non-GitHub) project and it no longer is part of `ts_ess_sensors`.